pkg/liqo-controller-manager/foreign-cluster-operator/foreign-cluster-controller.go

```diff
@@ -420,14 +450,23 @@ func (r *ForeignClusterReconciler) checkIncomingPeeringStatus(ctx context.Contex
 	return nil
 }
 
-// getResourceOffer returns the ResourceOffer for the given remote cluster, or a nil pointer if there is none.
+// getResourceOffer returns the incoming or outgoing ResourceOffer for the given remote cluster, or a nil pointer if
+// there is none.
 func (r *ForeignClusterReconciler) getResourceOffer(ctx context.Context,
```

Comment:
The name of the method should be explicitly said if it is dedicated to get the local or remote ResourceRequest. Ex GetLocalResourceOffer() / GetRemoteResourceOffer()